updatecli/updatecli.d/grafana.yaml

```
@@ -0,0 +1,53 @@
+sources:
+  lastRelease:
+    kind: helmchart
+    spec:
+      url: 'https://grafana.github.io/helm-charts'
+      name: 'grafana-agent-operator'
+
+targets:
+  chart:
+    name: Bump Chart Version
+    kind: yaml
+    spec:
+      file: 'grafana/kustomization.yaml'
+      key: '$.helmCharts[0].version'
+    transformers:
+    - addprefix: "'"
+    - addsuffix: "'"
+  kubectl:
+    name: run kubectl when chart changed
+    kind: shell
+    scmid: github
+    dependson:
+    - chart
+    dependsonchange: true
+    disablesourceinput: true
+    spec:
+      command: "rm -rf grafana/charts ksm/charts && kubectl kustomize . -o grafana-agent.yaml --enable-helm"
+      environments:
+      - name: PATH
+
+scms:
+  github:
+    kind: "github"
+    spec:
+      user: "argoyle"
+      email: "updatecli@opzkit.io"
+      owner: "opzkit"
+      repository: "terraform-aws-k8s-addons-grafana-agent-operator"
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      username: 'UpdateCLI'
+      branch: "main"
+
+# Define action configurations if one needs to be created
+actions:
+  addon:
+    kind: "github"
```

```suggestion
    kind: "github/pullrequest"
```

"github" is deprecated in favor of "github/pullrequest"